jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1784,7 +1784,7 @@ private static String getNonArrayClassPackageName(Class<?> clz) {
 /**
  * Answers the name of the package to which the receiver belongs.
  * For example, Object.class.getPackageName() returns "java.lang".
- * Returns null if this class represents an array type, a primitive type or void
+ * Returns "java.lang" if this class represents represents a primitive type or void, and the element type's package name in the case of an array type.
```

Comment:
Please remove the duplicate 'represents' and wrap the long line.

Reply:
Fixed the duplicated word.

Reply:
Line deliberately not wrapped per instructions from @DanHeidinga .